packages/@lwc/integration-karma/test/signal/protocol/index.spec.js

```diff
@@ -0,0 +1,95 @@
+import { createElement } from 'lwc';
+import Reactive from 'x/reactive';
+import NonReactive from 'x/nonReactive';
+import Parent from 'x/parent';
+import Child from 'x/child';
+import { Signal } from 'x/signal';
+
+describe('signal protocol', () => {
+    describe('lwc engine subscribes template re-render callback when signal is bound to an LWC and used on a template', () => {
```

Comment:
These tests validate when the framework will attempt to subscribe to a signal used in an LWC.